python-app/_pipenv/action.yml

```diff
@@ -0,0 +1,63 @@
+name: "test"
+description: "run unittest"
+
+inputs:
+  sync-dev:
+    description: "sync with --dev"
+    required: false
+    default: true
+
+runs:
+  using: "composite"
+  steps:
+    - name: clone repo
+      uses: actions/checkout@v3
```

Comment:
Since we clone, setup python, etc maybe we could make this action not a private one and call it `init`?
I can already see a couple locations I can use it where the rest of the steps (build/test) are more complex than standard.